docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -779,10 +791,30 @@ across all application deployments on the AS instance).
 |hibernate.auto_quote_keyword=false | 
 
 |hibernate.implicit_naming_strategy=org.hibernate.boot.model.naming.ImplicitNamingStrategyJpaCompliantImpl
+
+|hibernate.model.generator_name_as_sequence_name=true |
+
+|hibernate.jpa.compliance.transaction=true |
+
+|hibernate.jpa.compliance.closed=true |
+
+|hibernate.jpa.compliance.query=true |
+
+|hibernate.jpa.compliance.list=true |
+
+|hibernate.jpa.compliance.caching=true |
+
+|hibernate.jpa.compliance.proxy=true |
+
+|hibernate.enable_lazy_load_no_trans=false |
+
+|hibernate.jpa.compliance.global_id_generators=true |
+
+|hibernate.search.index_uninverting_allowed=true |
 | 
 |=======================================================================
 
-In Hibernate 4.x (and greater), if *new_generator_mappings* is *true*:
+if *new_generator_mappings* is *true*:
```

Comment:
Does "new_generator_mappings" mean "hibernate.id.new_generator_mappings"? If so, is the full property name listed somewhere? If not, the full name should be mentioned.

Reply:
yes, they are the same.  I updated the shorthand reference to hibernate.id.new_generator_mappings, to hibernate.id.new_generator_mappings.